docker-compose.yml

```diff
@@ -0,0 +1,37 @@
+version: '3.1'
+
+services:
+
+  # PostgreSQL database server
+  psql:
+    image: postgres
+    ports:
+      - 5432:5432
```

Comment:
```suggestion
```
Makes it able to use already install postgres together with docker postgres at the same machine